.github/workflows/static-code-analysis.yml

```diff
@@ -7,10 +7,6 @@ on:
     branches:
       - dev
 
-defaults:
```

Comment:
See my notes about the workflow here: https://github.com/Lombiq/Utility-Scripts/pull/28#discussion_r1052640912